reg_tests/test_files/ablNeutralEdgeNoSlip/README

```diff
@@ -0,0 +1,68 @@
+Neutral Atmospheric Boundary Layer Case
+
+30 Nov 2017
+
+This case is an example for running a large-eddy simulation of a neutral-
+stability atmospheric boundary layer.  The mesh resolution is much coarser 
+than would be run in practice, but that can be easily changed for a real 
+case.  The case should be run for at least 16,000 s to acheive a level
+of quasi equlibrium.  We recommend running this case with 10 m uniform
+resolution in all directions (it is set up for uniform 40 m resolution
+for this regression test).  The stability is neutral with no heat flux
+at the ground, but the boundary layer is capped with a stable inversion
+layer.  Because of constant density used here, because we do not include
+pressure work term in the enthalpy equation, and becuase we use Boussinesq
+buoyancy, temperature is potential temperature.
+
+- To build the mesh, use NaluWindUtils abl_mesh with nalu_abl_mesh.yaml
+- To initialze the fiels, use NaluWindUtils nalu_preprocess with 
+  nalu_preprocess.yaml
+
+Domain: 
+  - x: 0 m - 5000 m
+  - y: 0 m - 5000 m
+  - z: 0 m - 1000 m
+  - periodic laterally
+
+Winds:
+  - specified 8 m/s from 245 degrees (southwest) enforced with a
+    P-controlled driving source term.
+
+Initial temperature profile:
+  - 300 K constant to 650 m
+  - linear varying 300 K - 308 K from 650 m - 750 m
+  - linear increasing at a rate of 3 K per 1000 m above 750 m
+
+Initial velocity:
+  - Constant 8 m/s from 245 degrees throughout with sinusoidal
+    perturbations near the surface
+
+Surface:
+  - Rough wall shear stress model based on Monin-Obukhov similarity.
+  - z0 = 0.2 m
+  - qWall = 0 W/m^2
```

Comment:
Should we add an explanation of the _no-slip_ implementation here to differentiate this from the `ablNeutralEdge` case?

Reply:
Yeah good idea. Maybe after the first sentence add something like:

_This case utilizes the no-slip wall stress boundary condition proposed by Jaegle et al. in their 2010 paper "Implementation Methods of Wall Functions in Cell-vertex Numerical Solvers".  The rest of the setup is identical to the ablNeutralEdge regression test._ 

and then just delete the rest of the paragraph

Reply:
README was updated